cg/cli/workflow/mip_rna/base.py

```diff
@@ -0,0 +1,139 @@
+""" Add CLI support to start MIP rare disease RNA"""
+
+import logging
+
+import click
+
+from cg.apps import hk, tb, lims
+from cg.apps.environ import environ_email
+from cg.apps.mip import MipAPI
+from cg.apps.mip.fastq import FastqHandler
+from cg.cli.workflow.mip_rna.store import store as store_cmd
+from cg.cli.workflow.mip_rna.deliver import deliver as deliver_cmd
+from cg.cli.workflow.get_links import get_links
+from cg.meta.workflow.mip_rna import AnalysisAPI
+from cg.meta.deliver.mip_rna import DeliverAPI
+from cg.store import Store
+
+LOG = logging.getLogger(__name__)
+
+
+@click.group("mip-rna")
+@click.pass_context
+def mip_rna(context: click.Context):
+    """Rare disease RNA workflow"""
+    context.obj["db"] = Store(context.obj["database"])
+    hk_api = hk.HousekeeperAPI(context.obj)
+    lims_api = lims.LimsAPI(context.obj)
+    context.obj["tb"] = tb.TrailblazerAPI(context.obj)
+    deliver = DeliverAPI(context.obj, hk_api=hk_api, lims_api=lims_api)
+    context.obj["api"] = AnalysisAPI(
+        db=context.obj["db"],
+        hk_api=hk_api,
+        tb_api=context.obj["tb"],
+        lims_api=lims_api,
+        deliver_api=deliver,
+    )
+    context.obj["rna_api"] = MipAPI(
+        context.obj["mip-rd-rna"]["script"], context.obj["mip-rd-rna"]["pipeline"]
+    )
+
```

Comment:
Why aren't the subcommands invoked here as in the mip_dna command?

Reply:
This should definitely be done but it is a new feature and this PR is contained to mostly be a refactoring